docs/data/sources/google-tag-manager-client.md

```diff
@@ -3,17 +3,256 @@ title: Google Tag Manager Template - Amplitude Analytics Browser SDK
+This is the client-side Google Tag Manager Template for Amplitude Analytics. The tag uses the [Amplitude Marketing Analytics SDK](../../sdks/marketing-analytics-browser/) for data collection. However, due to inherent limitations of GTM, certain features, such as plugins, are not well supported in this GTM template. You are still able to add plugins using the Custom HTML tag, but because of how SDK get loaded GTM, this could lead missing data.
 
 !!!info Resources
     [:simple-googletagmanager: GTM Template Gallery](https://tagmanager.google.com/gallery/#/owners/amplitude/templates/amplitude-browser-sdk-gtm-template) · [:material-github: GitHub](https://github.com/amplitude/amplitude-browser-sdk-gtm-template)
 
-It’s important to create an `init` tag. It's common to fire the `init` tag using `All Pages` or `Initialization - All Pages` triggers. With this template, you can also defer the `init` tag until you receive a signal and fire it using customized triggers, such as a consent grant. But notice that all other tags wait for the `init` tag to fire before they can be sent to Amplitude. The legacy template automatically calls init, so there's no need to do it yourself.
+## Workflow
 
-This video tutorial walks through the implementation basics. 
+### Container Setup
+
+If you start from zero, you need to setup your conatiner first. This Amplitude Analytics Browser SDK tag template can be found in **Web** target platform which for the uses on descktop and mobile web pages.
+
+![Web Container Setup](../../assets/images/gtm/gtm-web-container-setup.png) 
+
+### Add Template
+
+Create a new tag template by searching the gallery. Choose Amplitude Analytics Browser SDK and click `add` button.
+![Amplitude Analytics Browser SDK ](../../assets/images/gtm/gtm-amplitude-analytics-browser-sdk.png) 
+
+### Create Tags
+
+Create tags for your amplitude browser SDK tracking. Click the `New` button to create your new tags.
+
+#### Tag Configuration
+
+Amplitude provides a list of tag types. You might also create a Custom HTML tag to do customization, like cross domain tracking or others.  For successful tracking in Amplitude, ensure that the Amplitude tag loads before the Custom HTML tag, which is being used to call Amplitude. The sequence in which the tags load can significantly influence the outcome. [Time sequence](https://support.google.com/tagmanager/answer/6238868) might be helpful in this case. Please check [this section](./#access-amplitude-instance) for accessing amplitude instance in the Custom HTML tag. Note: We are not recomended to install amplitude through Cusom HTML tag.
+ 
+#### API Key
+
+Copy your amplitude API Key in the API Key field. For EU recidency, please make your API Key is under analytics.eu.amplitude.com. See [this page](https://tagmanager.google.com/?authuser=0#/container/accounts/6053924196/containers/91603214/workspaces/62/tags:~:text=project%20settings.%20See-,this%20page,-for%20more%20information) for more details. 
```

Comment:
For accessibility I think "see" is not recommended terminology. I have been using "Learn more" or "More details" instead.

```suggestion
Copy your amplitude API Key in the API Key field. For EU recidency, please make your API Key is under analytics.eu.amplitude.com. Learn more [here](https://tagmanager.google.com/?authuser=0#/container/accounts/6053924196/containers/91603214/workspaces/62/tags:~:text=project%20settings.%20See-,this%20page,-for%20more%20information). 
```